app/utils/answersHelpers/AnswersHelper.scala

```diff
@@ -73,6 +73,23 @@ class AnswersHelper(userAnswers: UserAnswers)(implicit messages: Messages) exten
         )
     }
 
+  def buildRowWithNoChangeLink[T](
+    data: Option[T],
+    formatAnswer: T => Content,
+    prefix: String,
+    args: Any*
+  ): Option[SummaryListRow] =
+    data map {
+      answer =>
+        buildRow(
+          prefix = prefix,
+          answer = formatAnswer(answer),
+          id = None,
+          call = None,
+          args = args: _*
+        )
+    }
+
```

Comment:
Do we need this new method?
I believe we should be able to repurpose the already existing one instead.